installer/scripts/omsagent.systemd

```diff
@@ -6,7 +6,7 @@ Wants=omid.service
 [Service]
 Type=simple
 User=omsagent
-Group=omsagent
+Group=omiusers
```

Comment:
What sort of effect would that have if you're starting omsagent as an init.d service? I'd think you'd need to account for this change there as well?

Suggest you test on RH6 or SLES11 or something older than system to verify (if you didn't already).